dev/com.ibm.ws.microprofile.faulttolerance_fat_tck/fat/src/com/ibm/ws/microprofile/faulttolerance/tck/FaultToleranceTckPackageTest.java

```diff
@@ -51,83 +45,67 @@ public static void setUp() throws Exception {
+                          "CWMFT0014E", // CWMFT5014E: The CircuitBreaker policy parameter requestVolumeThreshold value
+                          // -1 for the
+                          // org.eclipse.microprofile.fault.tolerance.tck.invalidParameters.CircuitBreakerClientForValidationReqVolNeg.serviceA
+                          // is not valid, because the parameter must not be a negative number.
+                          "CWMFT0015E", // CWMFT5015E: The CircuitBreaker policy parameter successThreshold value 0 for
+                          // the
+                          // org.eclipse.microprofile.fault.tolerance.tck.invalidParameters.CircuitBreakerClientForValidationSuccessNeg.serviceA
+                          // is not valid, because the parameter must not be a negative number.
+                          "CWMFT0016E", // CWMFT5016E: The Bulkhead policy parameter value value -1....
+                          "CWMFT5017E", // CWMFT5017E: The Retry policy maximum duration of 500 Millis for the
+                          // org.eclipse.microprofile.fault.tolerance.tck.invalidParameters.RetryClientForValidationDelayDuration.serviceA
+                          // target is not valid as it must be greater than the delay duration of 1,000
+                          // Millis.
+                          "CWWKZ0002E");// CWWKZ0002E: An exception occurred while starting the application
+                                        // ftInvalidCB5. The exception message was:
+                                        // com.ibm.ws.container.service.state.StateChangeException:
+                                        // org.jboss.weld.exceptions.DefinitionException
     }
 
     /**
      * Run the TCK (controlled by autoFVT/publish/tckRunner/tcl/tck-suite.html)
-     * 
+     *
      * @throws Exception
      */
     @Test
     @AllowedFFDC // The tested exceptions cause FFDC so we have to allow for this.
     public void testTck() throws Exception {
-        if (!MvnUtils.init) {
-            MvnUtils.init(server);
-        }
-        // Everything under autoFVT/results is collected from the child build machine
-        // so we place the TCKs results there.
-        File mvnOutput = new File(MvnUtils.home, "results/mvnOutput_TCK");
-        int rc = MvnUtils.runCmd(MvnUtils.mvnCliTckRoot, MvnUtils.tckRunnerDir, mvnOutput);
-        File src = new File(MvnUtils.home, "results/tck/surefire-reports/junitreports");
-        File tgt = new File(MvnUtils.home, "results/junit");
-        try {
-            Files.walkFileTree(src.toPath(), new MvnUtils.CopyFileVisitor(src.toPath(), tgt.toPath()));
-        } catch (java.nio.file.NoSuchFileException nsfe) {
-            Assert.assertNull(
-                    "The TCK tests' results directory does not exist which suggests the TCK tests did not run - check build logs."
-                            + src.getAbsolutePath(),
-                    nsfe);
-        }
+        int rc = MvnUtils.runTCKMvnCmd(server);
```

Comment:
I had a version like yours but I chose to undo it as I did not want to standardise all the directory/file names in the common code in a way that made it harder to have multiple TCK FAT tests (results directories) co-oexisting in one FAT bucket. That is why these file names are here in the FAT in a single test and not in MvnUtils. I think what I had (and what you have done) looks neater and is easier to clone though. So I will approve it.